src/components/accordion/AccordionItem.tsx

```diff
@@ -34,6 +35,9 @@ const accordionIconMargin: IOSpacingScale = 8;
 const accordionBorder: IOColors = "grey-200";
 const accordionBackground: IOColors = "white";
 
+// Icon size
+const iconSize: IOIconSizeScale = 20;
```

Comment:
According to the design guidelines:
```suggestion
const iconSize: IOIconSizeScale = 24;
```